src/GafferSceneUI/TranslateTool.cpp

```diff
@@ -116,6 +116,12 @@ void TranslateTool::updateHandles()
 {
 	handles()->setTransform( handlesTransform() );
 
+	// Because we provide multiple orientations, the handles
+	// may well not be aligned with the axes of the transform
+	// space. So any given handle might affect several components
+	// of the target translation. For each handle, check to see
+	// if each of the plugs it effects are settable, and if not,
+	// disable the handle.
```

Comment:
Actually, it seems my test had a rotate in it, and that was throwing off what I was expecting to see. Lets ignore me for now.